util/java/libtiled-java/libtiled-java.qbs

```diff
@@ -0,0 +1,16 @@
+import qbs
+
+JavaJarFile {
+    files: ["src/**/*.java"]
+
+    Group {
+        fileTagsFilter: ["java.jar"]
+        qbs.install: true
+        qbs.installDir: "jar"
+    }
+
+    Export {
+        Depends { name: "java" }
+        java.manifestClassPath: [product.targetName + ".jar"]
```

Comment:
It's great that these .jar files can be compiled using Qbs, but it really shouldn't be required as part of building Tiled. Currently, on a system without a JDK installed, you get:
```
ERROR: /usr/share/qtcreator/qbs/share/qbs/modules/java/JavaModule.qbs:349 The following properties are not set. Set them in your profile or product:
java.jdkPath
java.compilerVersion
java.compilerVersionMajor
java.compilerVersionMinor
java.compilerVersionUpdate
The following properties have invalid values:
java.compilerVersion: must be a version number with 4 to 4 components
java.compilerVersionMajor: must be an integer >= 1
java.compilerVersionMinor: must be an integer >= 0
java.compilerVersionPatch: must be an integer >= 0
java.compilerVersionUpdate: must be an integer >= 0
```
So at the least it should auto-disable based on whether a Java compiler is available.

I'm currently undecided on whether this product should build by default at all, since it is not required by Tiled and it is low quality and out of date. Even if it is built by default, it should probably not have an install rule.